script_runner/frontend/src/Input.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from "react";
```

Comment:
Can we add tests for this component? I don't know if you have tests for frontend code or not, but it would be nice to add them. Frontend tests have actually caught issues for me in the past.

Reply:
we don't have at the moment. we need to figure out our approach for UI testing

personally i am not sure about component level tests, and unit tests in frontend in general. i wonder if integration or visual testing might be a better first place to start.

Reply:
Personally i like some test (even if its not perfect) over no tests. We can punt on this for now. But I would suggest we choose some mechanism in the near future and start rolling out tests.

Reply:
Fair. I can put together an example with a couple of styles soon and see what we like best.